packages/adminrouter/extra/dcos-adminrouter.service

```diff
@@ -23,6 +23,6 @@ ExecStartPre=/usr/bin/test -f /var/lib/dcos/cluster-id
 # Wait for auth backend to seed the secret key
 ExecStartPre=/usr/bin/curl --fail -sS -o /dev/null 127.0.0.1:8101/acs/api/v1/groups
 ExecStart=$PKG_PATH/nginx/sbin/nginx -c $PKG_PATH/nginx/conf/nginx.master.conf
-ExecReload=/usr/bin/kill -HUP $MAINPID
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
1. Sure - do we have a list of officially supported distros and versions? I'll pull some dockers down and confirm that /bin/kill exists in all of them.
2. Thanks for pointing that out - I didn't know they suggest using a synchronous reload option if possible. In this case, the official `nginx -s reload` command just sends SIGHUP to the server, too.